lms/djangoapps/grades/models.py

```diff
@@ -121,6 +123,7 @@ def create_from_blockrecords(self, blocks):
         """
         model, _ = self.get_or_create(
             hashed=blocks.hash_value,
+            version=VISIBLE_BLOCKS_VERSION,
```

Comment:
It will do an indexed query, but it won't use the version field when doing the query.  For this index, I think it's okay, since the hashes should be globally unique anyway.